### DISCUSSION TITLE
Welcome to Talk Python in Production Book Discussions!

### DISCUSSION THREAD

**Discussion by mikeckennedy** [mikeckennedy/talk-python-in-production-devops-book#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by possnfiffer** [comment]
Hello everyone,

My name is Roller Angel, and I’m a Senior Systems Administrator who loves building projects with Python on FreeBSD. I’ve been learning with Talk Python for years — Michael’s insights and teaching style have been a huge inspiration.

I just picked up the Talk Python in Production book and plan to follow along closely. My setup will be a little different: I’m running a cloud VM as a proxy and VPN endpoint that connects back to my home lab, which has 8 Gb fiber and plenty of horsepower for my FreeBSD jails. I’ll also be launching a bhyve VM running Linux so I can mirror the book’s environment before adapting it to my own infrastructure.

I work a lot with FastAPI, HTMX, Chameleon, and MongoDB, currently deployed under Gunicorn. I haven’t yet switched to the WSGI server used in the book (or to uv), but I’m looking forward to exploring both.

On the tooling side, I’ve been diving into Zsh lately. I’ve long been a tcsh fan, but after getting a new MacBook Air M4, I’m embracing Zsh as my daily shell. My workflow revolves around SSH’ing into my FreeBSD lab from the Air, and I really enjoy the mix of PyCharm and the Apple ecosystem for development.

I’ve been a DIY tinkerer for years — I used to flash custom Android ROMs (most recently GrapheneOS) before moving fully into Apple land with an iPhone Air, Apple Watch, AirPods, Apple TV, and the MacBook Air. Honestly, the seamless integration still surprises me; so many things that used to take effort now just work.

I’m excited to see what everyone else is building with this book, and I’ll share updates on my own projects as I go. My long-term goal is to build a BSD training website to showcase all the cool things you can do with FreeBSD and Python.

Looking forward to learning and sharing with you all!
— Roller
